app/views/content/blog/choosing-the-right-teacher-training-course-provider.md

```diff
@@ -25,7 +25,7 @@ When choosing a course provider for your teacher training, it’s important to c
 
 Your drive to become a teacher needs to inform your choice of provider. If you have a particular interest, make sure that the school placements you will be offered for your teaching practice will allow you to develop it. For example, there are providers that work with fully comprehensive and inclusive schools, faith-based schools, special schools, grammar schools, schools in challenging social or economic areas, rural schools and so on.
 
-You can find the available postgraduate teacher training courses in your area via the [Department for Education’s Find service](https://www.gov.uk/find-postgraduate-teacher-training-courses). Consider selecting 4-5 that you feel might be suitable for you, and then make contact with them to check that they are. Aim to shortlist these down to 3. 
+Take a look at the available [postgraduate teaching training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in your area. Consider selecting 4-5 that you feel might be suitable for you, and then make contact with them to check that they are. Aim to shortlist these down to 3. 
```

Comment:
Although we're not as wedded to it as other services, we try to follow GDS styles where we can. For this line, GDS would advise that we start with a stronger action, and always with as few words as possible. 

With that in mind, would change to "_Search_ the available..."